src/Microsoft.ML.Data/Transforms/ValueToKeyMappingTransformer.cs

```diff
@@ -804,7 +804,7 @@ private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, ColInfo info, string src
                 else if (info.TypeSrc.GetItemType().Equals(NumberDataViewType.Double))
                 {
                     // LabelEncoder doesn't support double tensors, so values are cast to floats
-                    var castOutput = ctx.AddIntermediateVariable(null, "castOutput", true);
```

Comment:
>null [](start = 65, length = 4)

How did it work with null before? Was there an exception?

Reply:
If the last parameter is true, it skips adding shape and type information and therefore accepts null. This still works, but I am prepping some parts of the code base for issues I have seen when run against the master branch of ORT. 

---
In reply to: [383088824](https://github.com/dotnet/machinelearning/pull/4878#discussion_r383088824) [](ancestors = 383088824)